packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -238,8 +238,8 @@ export class AtomicSearchBox {
 
   private get isDoubleList() {
     const numberOfSuggestionsLists =
-      (this.leftSuggestions.length ? 1 : 0) +
-      (this.rightSuggestions.length ? 1 : 0);
+      (this.leftSuggestionElements.length ? 1 : 0) +
+      (this.rightSuggestionElements.length ? 1 : 0);
 
     return numberOfSuggestionsLists === 2;
```

Comment:
seems like a bit os a convoluted way to do
```js
private get isDoubleList() {
    return Boolean(this.leftSuggestionElements.length && this.rightSuggestionElements.length)
```

Reply:
oh right, I converted this code from something like listsCount to isDoubleList and went too quick 🤦